src/components/NewWorkspaceModal.js

```diff
@@ -371,6 +373,27 @@ const NewWorkspaceModal = withDisplayName(
                     }),
                   ]),
               ]),
+              isGoogleBillingProject() &&
+                h(IdContainer, [
```

Comment:
To fix the a11y issue about a label, use this.

I'm somewhat conflicted about having the InfoBox popup within the label… pros and cons either way.

```
isGoogleBillingProject() &&
                h(Fragment, [
                  div({ style: { padding: '.25rem' } }),
                  h(LabeledCheckbox, {
                    checked: enhancedBucketLogging,
                    disabled: requireEnhancedBucketLogging || groups.length > 0,
                    onChange: () => setEnhancedBucketLogging(!enhancedBucketLogging),
                    children: [
                      h(FormLabel, { style: { paddingLeft: '0.5rem', display: 'inherit' } }, [
                        'Enhanced Bucket Logging',
                        h(InfoBox, { style: { marginLeft: '0.25rem', padding: '0' } }, [
                          '"Enhanced Bucket Logging" means ...',
                          h(
                            Link,
                            {
                              href: 'https://youtu.be/E4WlUXrJgy4',
                              ...Utils.newTabLinkProps,
                            },
                            ['Read more about Enhanced Bucket Logging']
                          ),
                        ]),
                      ]),
                    ],
                  }),
                ]),